packages/intent/src/commands/install.ts

```
@@ -1,5 +1,31 @@
+import { relative } from 'node:path'
+import { fail } from '../cli-error.js'
+import {
+  printWarnings,
+  scanOptionsFromGlobalFlags,
+  type GlobalScanFlags,
```

ESLint

[error] Member 'GlobalScanFlags' of the import declaration should be sorted alphabetically.

(sort-imports)